scripts/commands/checkLinks.ts

```diff
@@ -143,12 +143,13 @@ async function determineCurrentDocsFileBatch(
     "!public/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/dev/*",
   ];
   const toLoad = [
-    "docs/api/qiskit/0.44/{algorithms,opflow}.md",
-    "docs/api/qiskit/0.44/qiskit.{algorithms,extensions,opflow}.*",
-    "docs/api/qiskit/0.44/qiskit.utils.QuantumInstance.md",
+    // The 0.44 docs are used by release notes for APIs that were removed in 1.0.
+    "docs/api/qiskit/0.44/*.md",
     "docs/api/qiskit/0.45/qiskit.quantum_info.{OneQubitEuler,TwoQubitBasis,XX}Decomposer.md",
     "docs/api/qiskit/0.45/qiskit.transpiler.synthesis.aqc.AQC.md",
     "docs/api/qiskit/0.45/{tools,quantum_info,synthesis_aqc}.md",
+    "docs/api/qiskit/0.45/qiskit.passmanager.FlowController.md",
+    "docs/api/qiskit/0.45/qiskit.circuit.CommutationChecker.md",
   ];
```

Comment:
These changes are telling our link checker that it needs to read in these files because other files link to them.